| Software                                                                             | Version                               | Swset        | Architecture   | Category       | Clusters    | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
|:-------------------------------------------------------------------------------------|:--------------------------------------|:-------------|:---------------|:---------------|:------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <p><a href=https://www.htslib.org/>SAMtools</a></p>                                  | <p>1.12-GCC-10.2.0</p>                | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | SAM Tools provide various utilities for manipulating alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format.                                                                                                                                                                                                                                                                                                                                     |
| <p><a href=https://www.htslib.org/>HTSlib</a></p>                                    | <p>1.12-GCC-10.2.0</p>                | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | A C library for reading/writing high-throughput sequencing data. This package includes the utilities bgzip and tabix                                                                                                                                                                                                                                                                                                                                                                                            |
| <p><a href=https://www.gromacs.org>GROMACS</a></p>                                   | <p>2021-foss-2020b</p>                | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. This is a CPU only build, containing both MPI and threadMPI builds for both single and double precision. It also contains the gmxapi extension for the single precision MPI build.                                                                                                                                                            |
| <p><a href=https://www.bioinformatics.babraham.ac.uk/projects/fastqc/>FastQC</a></p> | <p>0.11.9-Java-11</p>                 | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline.                                                                                                                                                                                                    |
| <p><a href=https://www.bcgsc.ca/platform/bioinfo/software/abyss>ABySS</a></p>        | <p>2.2.5-foss-2020b</p>               | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler                                                                                                                                                                                                                                                                                                                                                                                                                                |
| <p><a href=https://github.com/pezmaster31/bamtools>BamTools</a></p>                  | <p>2.5.1-GCC-10.2.0</p>               | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.                                                                                                                                                                                                                                                                                                                                                                                                                     |
| <p><a href=https://blast.ncbi.nlm.nih.gov/>BLAST+</a></p>                            | <p>2.11.0-gompi-2020b</p>             | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences.                                                                                                                                                                                                                                                                                                 |
| <p><a href=https://bioperl.org/>BioPerl</a></p>                                      | <p>1.7.8-GCCcore-10.2.0</p>           | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | Bioperl is the product of a community effort to produce Perl code which is useful in biology. Examples include Sequence objects, Alignment objects and database searching objects.                                                                                                                                                                                                                                                                                                                              |
| <p><a href=https://bedtools.readthedocs.io/>BEDTools</a></p>                         | <p>2.30.0-GCC-10.2.0</p>              | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | BEDTools: a powerful toolset for genome arithmetic. The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.                                                                                                                                                                                                                                |
| <p><a href=http://ccb.jhu.edu/software/tophat/>TopHat</a></p>                        | <p>2.1.2-GCC-10.2.0-Python-2.7.18</p> | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | TopHat is a fast splice junction mapper for RNA-Seq reads.                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
| <p><a href=http://bowtie-bio.sourceforge.net/bowtie2/index.shtml>Bowtie2</a></p>     | <p>2.4.2-GCC-10.2.0</p>               | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes. |
| <p><a href=http://bio-bwa.sourceforge.net/>BWA</a></p>                               | <p>0.7.17-GCC-10.2.0</p>              | <p>2020b</p> | <p>epyc</p>    | <p>Biology</p> | <p>aion</p> | Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively short nucleotide sequences against a long reference sequence such as the human genome.                                                                                                                                                                                                                                                                                                                                             |