| Software                                                              | Version                               | Swset        | Architecture   | Category       | Clusters    | Description                                                                                                                                                                                                                                                                         |
|:----------------------------------------------------------------------|:--------------------------------------|:-------------|:---------------|:---------------|:------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <p><a href=https://trinityrnaseq.github.io>Trinity</a></p>            | <p>2.10.0-foss-2019b-Python-3.7.4</p> | <p>2019b</p> | <p>skylake</p> | <p>Biology</p> | <p>iris</p> | Trinity represents a novel method for the efficient and robust de novo reconstruction of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm, Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads. |
| <p><a href=https://pachterlab.github.io/kallisto/>kallisto</a></p>    | <p>0.46.1-foss-2019b</p>              | <p>2019b</p> | <p>skylake</p> | <p>Biology</p> | <p>iris</p> | kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads.                                                                                                                    |
| <p><a href=https://github.com/COMBINE-lab/salmon>Salmon</a></p>       | <p>1.1.0-gompi-2019b</p>              | <p>2019b</p> | <p>skylake</p> | <p>Biology</p> | <p>iris</p> | Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data.                                                                                                                                                          |
| <p><a href=http://www.gromacs.org>GROMACS</a></p>                     | <p>2020-foss-2019b</p>                | <p>2019b</p> | <p>skylake</p> | <p>Biology</p> | <p>iris</p> | GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. This is a CPU only build, containing both MPI and threadMPI builds.                                               |
| <p><a href=http://www.genome.umd.edu/jellyfish.html>Jellyfish</a></p> | <p>2.3.0-GCC-8.3.0</p>                | <p>2019b</p> | <p>skylake</p> | <p>Biology</p> | <p>iris</p> | Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.                                                                                                                                                                                                           |
| <p><a href=http://www.bioperl.org/>BioPerl</a></p>                    | <p>1.7.2-GCCcore-8.3.0</p>            | <p>2019b</p> | <p>skylake</p> | <p>Biology</p> | <p>iris</p> | Bioperl is the product of a community effort to produce Perl code which is useful in biology. Examples include Sequence objects, Alignment objects and database searching objects.                                                                                                  |